src/pire.mk

```diff
@@ -0,0 +1,40 @@
+# This file is part of MXE.
+# See index.html for further information.
+
+PKG             := pire
+$(PKG)_IGNORE   :=
+$(PKG)_VERSION  := 0.0.5
+$(PKG)_SOVERS   := 0
+$(PKG)_CHECKSUM := fc5f451043c6fc1034f104463a4b9c6c1c46c00c
+$(PKG)_SUBDIR   := pire-release-$($(PKG)_VERSION)
+$(PKG)_FILE     := release-$($(PKG)_VERSION).tar.gz
```

Comment:
This URL was [produced](https://github.com/yandex/pire/releases) by Github: `https://github.com/yandex/pire/archive/release-0.0.5.tar.gz` for tag `release-0.0.5`.